osbs/core.py

```diff
@@ -43,14 +43,17 @@
 logger = logging.getLogger(__name__)
 
 
-def check_response(response):
+def check_response(response, log_errors=True):
     if response.status_code not in (httplib.OK, httplib.CREATED):
         if hasattr(response, 'content'):
             content = response.content
         else:
             content = b''.join(response.iter_lines())
 
-        logger.error("[%d] %s", response.status_code, content)
```

Comment:
> the level int for error is 40 and debug 10, isn't very readable

You wouldn't write `check_response(response, log_level=10)`, you'd write `check_response(response, log_level=logging.DEBUG)` which to me is very readable.

I'm fine as is, just thought it might be a little bit nicer to make it more flexible.